syscall/syscall.c

```diff
@@ -1067,6 +1068,34 @@ OE_WEAK OE_DEFINE_SYSCALL2(SYS_umount2)
     return oe_umount(target);
 }
 
+OE_WEAK OE_DEFINE_SYSCALL3_M(SYS_getrandom)
+{
+    oe_errno = 0;
+    long ret = -1;
+    void* buf = (void*)arg1;
+    size_t buflen = (size_t)arg2;
+    unsigned int flags = (unsigned int)arg3;
+
+    (void)flags;
```

Comment:
Would it be better to fail if flags != 0 instead of silently ignoring them?

Reply:
Just realize that the normal use of libc API may to set the flags. Error out would prevent such programs.

Reply:
actually I would claim that not implementing the behavior of the flags may prevent such programs.  Unless you have tested them, we should not claim to support the flags being set.

Reply:
Fix by #3934